examples/mlmirror.py

```diff
@@ -10,6 +10,13 @@
 Note that string1 and string2 depend on the paths of those files.
 '''
 
+def normalEnergyFunc(mean, stdDev):
+    def energyFunc(photonTimes):
```

Comment:
I think this deserves a short comment. I first thought that you mean "normal" energy in the sense of "what we usually do", but then I saw the "normal" means "Normal distribution".
Great use of a function generating a function, but that might also need a short explanation of what's going on here, if e.g. Hermann whats to use this later. ;-)